<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StagingWidget</class>
 <widget class="QWidget" name="StagingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Staging Management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="toolbarLayout">
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>üîÑ Refresh</string>
       </property>
       <property name="toolTip">
        <string>Refresh the list of staged batches</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>‚ùå Remove Selected</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Remove selected items from staging queue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteAllButton">
       <property name="text">
        <string>üóëÔ∏è Delete All Staged</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Permanently delete all staged batches</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #d32f2f; color: white; font-weight: bold; padding: 6px 12px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="emptyButton">
       <property name="text">
        <string>üóëÔ∏è Empty Old Batches...</string>
       </property>
       <property name="toolTip">
        <string>Permanently delete old batches with retention policies</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="batchTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path / Album</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quality Info</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="summaryLabel">
     <property name="text">
      <string>No staged batches</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
